nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -253,11 +253,6 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-schema-registry-service-api</artifactId>
```

Comment:
Is nifi-schema-registry-service-api intentionally removed here?

Reply:
It was duplicated in the pom, once as a test dep and once as a regular dep. Maven warned me about it.

Reply:
Again, a great change in a different PR for a different ticket.